src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureUnitTests.cs

```diff
@@ -890,5 +891,655 @@ static void Main(string[] args)
             test.ExpectedDiagnostics.Add(VerifyCS.Diagnostic(DetectPreviewFeatureAnalyzer.GeneralPreviewFeatureAttributeRule).WithLocation(0).WithArguments("SampleEvent"));
             await test.RunAsync();
         }
+
+        [Fact]
+        public async Task TestTypeOf()
+        {
+            var csInput = @" 
+using System.Runtime.Versioning; using System;
+namespace Preview_Feature_Scratch
+{" +
+@"
```

Comment:
Here and everywhere else why need to concatenate strings?
```suggestion
{
```

Reply:
There used to be another variable being appended between the 2 parts. I got rid of that variable in the last PR, but just kept this around. Fixed now. Removed the `+`